randip/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/fatih/color"
+	"github.com/ipinfo/cli/lib"
+	"github.com/ipinfo/cli/lib/complete/install"
+	"github.com/spf13/pflag"
+)
+
+var progBase = filepath.Base(os.Args[0])
+var version = "1.0.0"
+
+func printHelp() {
+	fmt.Printf(
+		`Usage: %s [<opts>] 
+
+Description:
+  Generates random IP/IPs.
+  Generates 1 IPv4 without providing any arguments. 
```

Comment:
See the fixes I recommended for the help message in `cmd_randip` - they should be applied here too.